ui/admin/tests/acceptance/roles/global-scope-test.js

```diff
@@ -138,7 +138,9 @@ module('Acceptance | roles | global-scope', function (hooks) {
     await waitUntil(
       () =>
         findAll(GRANT_SCOPE_ROW_SELECTOR(instances.role.grant_scope_ids[1]))
-          .length === 0,
```

Comment:
two test cases started failing. im thinking its because a reload causes the loading state to momentarily appear which was causes the waitUntil to pass without actually waiting for the search request to complete.

Reply:
I played around with this a bit this morning and I think you are right. Seemed to be jumpy a bit with the loading screen.